src/routes/klageskjema/begrunnelse/begrunnelse.tsx

```diff
@@ -191,17 +193,34 @@ const Begrunnelse = ({ klage }: Props) => {
                 </KlageAlertStripeFeil>
             )}
 
-            <Reasons checkedReasons={reasons} setCheckedReasons={setReasons} />
-
             <Section>
-                <KlageUndertittel>Vedtaksdato (valgfri)</KlageUndertittel>
-                <Datepicker
-                    onChange={(dateISO, isValid) => setVedtakDate(isValid ? dateISO : null)}
-                    value={vedtakDate ?? undefined}
-                    showYearSelector
-                    limitations={{
-                        maxDate: new Date().toISOString().substring(0, 10)
-                    }}
+                <KlageUndertittel>Hva er du uenig i?</KlageUndertittel>
+
+                <Row>
+                    <Reasons checkedReasons={reasons} setCheckedReasons={setReasons} />
+                </Row>
+                <Row>
+                    <label htmlFor="inputId">
```

Comment:
`for` på `<label>` må ha samme verdi som `id` på `<input>` for å koble de sammen :)

`inputId` er bare navnet på egenskapen til `Datepicker`.

Reply:
> `for` på `<label>` må ha samme verdi som `id` på `<input>` for å koble de sammen :)
> 
> `inputId` er bare navnet på egenskapen til `Datepicker`.

x) lol.. fiksa